src/commands/deployAzurePipeline/configureCicdPipeline/models/aksClusterNode.ts

```diff
@@ -0,0 +1,13 @@
+import { CloudExplorerV1 } from "vscode-kubernetes-tools-api";
+
+export class aksClusterNode{

```

Comment:
We should not have `node` in the name as this definition's purpose is to decouple tree node from the cluster object which DTA extension understands